packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -364,7 +369,7 @@ export function s(
     data: ElementCompilerData,
     children: VNodes,
     slotset: SlotSet | undefined
-): VElement {
+): VElement | VNodes {
```

Comment:
It's a bit odd to me that we're using `VNodes` here instead of `VElements`, but I guess this is because `SlotSet` is a map where the values are `VNodes`?

Reply:
right, imagine that the content of the slot (default content) is just a text node. In that case, it is perfectly valid, and should contain VNodes